generator/uscore/metadata_extractor.rb

```diff
@@ -0,0 +1,251 @@
+# frozen_string_literal: true
+
+module Inferno
+  module Generator
+    module USCoreMetadataExtractor
+      PROFILE_URIS = Inferno::ValidationUtil::US_CORE_R4_URIS
+
+      def profile_uri(profile)
+        "http://hl7.org/fhir/us/core/StructureDefinition/#{profile}"
+      end
+
+      def profile_json_uri(profile)
+        "https://www.hl7.org/fhir/us/core/StructureDefinition-#{profile}.json"
+      end
+
+      def search_param_path(resource, param)
+        param = 'id' if param == '_id'
+        "SearchParameter/us-core-#{resource.downcase}-#{param}"
+      end
+
+      def extract_metadata
+        metadata = {
```

Comment:
Do you know if they plan on updating the json for the capability statement and other definitions? The one offered from the site is still missing a lot of stuff.

Reply:
Grab all resources from the github repo directly.  For other things missing, we need to follow up on zulip.  You created tickets for this?